src/tools/rbd_mirror/instance_watcher/Types.cc

```diff
@@ -166,6 +203,15 @@ void NotifyMessage::decode(bufferlist::const_iterator& iter) {
   case NOTIFY_OP_SYNC_START:
     payload = SyncStartPayload();
     break;
+  case NOTIFY_OP_GROUP_ACQUIRE:
```

Comment:
Should 
DECODE_START(2, iter); 
be modified to 
DECODE_START(3, iter); 
as new notify types are being addeD?